### PR TITLE
Remove //tools.

### PR DESCRIPTION
**Pull request by chinmaygarde**
None of these except remove_stale_pyc_files seems to be used. But we have the
pyc files in the .gitignore and I couldn't find any existing checked in pyc
files. So this check has never made sense for the Flutter. Removing instead of
migrating.

Engine patch https://github.com/flutter/engine/pull/47032

### CODE REVIEW

**Review by matanlurey — APPROVED** [review]
LGTM

**Review by cbracken — APPROVED** [review]

File: tools/dart/create_updated_flutter_deps.py [hunk 1]
```diff
@@ -1,143 +0,0 @@
-#!/usr/bin/env python3
```
Comment (zanderso) [inline tools/dart/create_updated_flutter_deps.py:1]: This file is used by the Dart -> Engine autoroller.

Comment (chinmaygarde) [reply tools/dart/create_updated_flutter_deps.py:1]: Where are the autoroller sources?

Comment (zanderso) [reply tools/dart/create_updated_flutter_deps.py:1]: I don't remember where these live. Maybe @erock2112 Could help us track this down? Might have to discuss in the internal chat.

**Comment by whesse** [comment]
create_updated_flutter_deps.py is also used by the monorepo roller, by the script https://dart.googlesource.com/monorepo/+/refs/heads/main/tools/create_monorepo_deps.dart#39

**Comment by chinmaygarde** [comment]
That script was restored in https://github.com/flutter/buildroot/commit/ab76ccba2ca4ce40c7f52f9007704906d0bb5362. Would it be possible to move the script into the repo for the monoroller? It is hard to the test a free floating script that doesn't seem to be used in the repo. We are in the middle of migrating off of the buildroot so files might be shuffled around a bit.

**Pull request merged**